docs/Makefile

```diff
@@ -38,10 +40,10 @@ sphinx_build_dir: clean
 	@mkdir -p $(BUILDDIR)
 
 html/tt-metalium: build_doxygen sphinx_build_dir
-	@DOCS_VERSION=$(DOCS_VERSION) REQUESTED_DOCS_PKG=tt-metalium $(SPHINXBUILD) "$(SOURCEDIR)/tt-metalium" "$(TT_METALIUM_BUILDDIR)" $(SPHINXOPTS) $(O)
+	@DOCS_VERSION=$(DOCS_VERSION) REQUESTED_DOCS_PKG=tt-metalium $(SPHINXBUILD) "$(SOURCEDIR)/tt-metalium" "$(TT_METALIUM_BUILDDIR)"
 
 html/ttnn: build_doxygen sphinx_build_dir
-	@DOCS_VERSION=$(DOCS_VERSION) REQUESTED_DOCS_PKG=ttnn $(SPHINXBUILD) "$(SOURCEDIR)/ttnn" "$(TTNN_BUILDDIR)" $(SPHINXOPTS) $(O)
+	@DOCS_VERSION=$(DOCS_VERSION) REQUESTED_DOCS_PKG=ttnn $(SPHINXBUILD) "$(SOURCEDIR)/ttnn" "$(TTNN_BUILDDIR)"
 
```

Comment:
A little out of context. Why do you remove those options?

Reply:
Until we fix the docstrings (remove signatures from the first line, there will be warnings now coming from sphinx, so I had to remove the flag that turns warnings into errors.

Reply:
Would it be worth commenting out instead with a mention of this?

Reply:
+1 for Evan's comment

Reply:
EYON*